docs/user-documentation/getting-started-with-islandora-claw.md

```diff
@@ -0,0 +1,76 @@
+# Getting Started with Islandora CLAW
+
+## How to Add a User
+1. Log in under admin credentials.
+2. Click the **Manage** tab from the top toolbar.
+3. Select **People** tab from the resulting menu.
+ ![Alt text](../assets/people_page.png "People page")
+4. Enter all the required fields, as indicated by red asterisks, in the *Add User* page.
+5. Click on the **Create New Account** button at the bottom of the page.
+6. To review/edit the permission for each role, click the **Permissions** tab in the set of tabs above the *Add User* button.
+
+## How to Create a New User Role
+1. Click the **Manage** tab from the top toolbar.
+2. Select **People** tab from the resulting menu.
+3. Click the **Permissions** tab in the set of tabs above the *Add User* button.
+4. Click the **Add role** button 
+ ![Alt text](../assets/gettingstarted_roles.png "Roles page")
+5. Name the role in a way that it can be disambiguated from related activities or similar groups
+    * I.e. use the course code for a particular class of students working on a particular collection.
+6. Click the **Save** button.
+7. To edit the permissions for a role, click on the **Edit** dropdown menu to the right of the role and select **Edit Permissions**.
+8. Scroll down until you see the *Islandora section* of options. (Currently this is the very bottom of a very long page)
+    * Resource ‘type’ is what you’ll most often deal with
```

Comment:
@ysuarez Recapping our [IRC conversation](http://irclogs.islandora.ca/2017-06-02.html) (starting @ 13:32). The rabbit hole is once more permissions get added, keeping a screenshot of them could be problematic.

However, currently there is only one section of permissions and as this is a **Getting Started with Islandora CLAW** document, maybe a single screenshot with some permissions is good enough. :man_shrugging: 

Here is some of the current permissions lines now using Nodes.
![islandora_permission](https://cloud.githubusercontent.com/assets/2857697/26738324/50d98280-4793-11e7-8fb1-a18aec8c40aa.jpg)